core/haxlib.vcxproj

```diff
@@ -180,6 +180,7 @@
     <NASM Include="emulate_ops.asm">
       <FileType>Document</FileType>
     </NASM>
+    <NASM Include="ia32.asm" />

```

Comment:
nit: So we don't really need to specify the `FileType` tag? Then shall we just delete line 181?

Reply:
Not sure what caused the `<FileType>` tag to be added, but it's unnecessary. It's deleted now.